CIP-Draft-Initial-Params/Initial.md

```diff
@@ -0,0 +1,172 @@
+| rho                   	| 3.0e-3                                                                	| Monetary expansion rate per epoch.  Governs the rewards that are returned from reserves to the ecosystem (treasury, stake pools and delegators).                                                                                  |
+| | |
+| poolDeposit           	| 500000000                                                              	| Pool deposit (in lovelace)                                                                                                                                                                                                       	|
+| keyDeposit            	| 2000000                                                                	| Deposit charged for stake keys (in Lovelace).  Ensures that unused keys are returned, so freeing resources.                                                                                                                      	|
+| | |
+| minFeeB               	| 155381                                                                 	| Base transaction fee (in lovelace).                                                                                                                                                                                              	|
+| minFeeA               	| 44                                                                     	| Additional transaction fee per byte of data (in lovelace).                                                                                                                                                                       	|
+| | |
+| minUTxOValue          	| 1000000                                                                	| Minimum allowed value in a UTxO.  Security-related parameter used to prevent the creation of many small UTxOs that could use excessive resource to process.                                                                      	|
+| | |
+| extraEntropy          	| ```{         "tag": "NeutralNonce"     }```                            	| Should additional entropy be included in the initial phases.  This provides additional certainty that the blockchain has not been compromised by the seed key holders.  Redundant once the system is sufficiently decentralised. 	|
+| eMax                  	| 18                                                                     	| Maximum number of epochs within which a pool can be announced to retire, starting from the next epoch.                                                                                                                 	|
+
```

Comment:
It would be valuable to also mention which of those parameters are updatable and which ones aren't

Reply:
It's actually only the updatable ones it seems already. so I'll frame this differently, what about also mentioning the non-updatable ones from the genesis?

Reply:
The auditors were concerned with those that could be updated.  We could add a section on genesis parameters

Reply:
Added a section on non-updatable parameters (note that there is at least one parameter hat is implicit).